mindjunkies/dashboard/tests/test_mixin.py

```
@@ -0,0 +1,58 @@
+
+    def get(self, request):
+        return HttpResponse("Success")
+
+
+@pytest.fixture
+def factory():
+    return RequestFactory()
+
+
+@pytest.mark.django_db
+@override_settings(
```

Ruff (0.8.2)

Undefined name `override_settings`

(F821)